src/components/NewRuntimeModal.js

```diff
@@ -328,7 +327,7 @@ export const NewRuntimeModal = withModalDrawer({ width: 675 })(class NewRuntimeM
         [(viewMode !== 'deleteEnvironmentOptions'), () => {
           return {
             cloudService,
-            toolDockerImage: _.includes(selectedLeoImage, [CUSTOM_MODE, PROJECT_SPECIFIC_MODE]) ? customEnvImage : selectedLeoImage,
+            toolDockerImage: _.includes(selectedLeoImage, [CUSTOM_MODE]) ? customEnvImage : selectedLeoImage,
```

Comment:
since it's only comparing the `selectedLeoImage` to `CUSTOM_MODE` now it might be simpler to use a `===` rather than `_.includes`